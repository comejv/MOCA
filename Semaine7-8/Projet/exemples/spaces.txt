test    yes 

  

or not yes
