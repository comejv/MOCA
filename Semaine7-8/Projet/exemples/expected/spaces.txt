not 1
or 1
test 1
yes 2
